### PR TITLE
2025 beta

### PR DESCRIPTION
**Pull request by Ishan1522**
currently will not build.

adds units and measures

### CODE REVIEW

**Comment by Ishan1522** [comment]
yay checks pass

File: src/main/java/frc/robot/subsystems/swerve/SwerveModule.java [hunk 1]
```diff
@@ -3,16 +3,12 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
-import edu.wpi.first.math.util.Units;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+// import edu.wpi.first.wpilibj.Alert;
```
Comment (JacksonElia) [inline src/main/java/frc/robot/subsystems/swerve/SwerveModule.java]: delete stuff like this

File: src/main/java/frc/robot/subsystems/swerve/SwerveModule.java [hunk 2]
```diff
@@ -22,38 +18,33 @@ public class SwerveModule extends SubsystemBase {
...
+  // private final Alert hardwareFaultAlert;
 
   public SwerveModule(ModuleInterface io, String name) {
     super("Module-" + name);
     this.io = io;
     this.name = name;
-    this.hardwareFaultAlert =
-        new Alert("Module-" + name + " Hardware Fault", Alert.AlertType.ERROR);
-    this.hardwareFaultAlert.setActivated(false);
+    // this.hardwareFaultAlert =
```
Comment (JacksonElia) [inline src/main/java/frc/robot/subsystems/swerve/SwerveModule.java]: delete

File: src/main/java/frc/robot/subsystems/swerve/gyroIO/GyroInterface.java [hunk 3]
```diff
@@ -1,13 +1,16 @@
 package frc.robot.subsystems.swerve.gyroIO;
```
Comment (JacksonElia) [inline src/main/java/frc/robot/subsystems/swerve/gyroIO/GyroInterface.java:1]: I like the names you chose for interface vs sim vs physical a lot. Very clear immediately. Make sure  you're moving docstrings to the interface.

**Review by JacksonElia — APPROVED** [review]
LGTM overall. Maybe clean up comments here and there if possible. Once this is merged, tmrw night perhaps, I'm gonna make a pr just cleaning stuff up and doing nits just so I can learn the code

**Pull request merged**